### PR TITLE
feat: add wishlist page

### PR DESCRIPTION
**Pull request by kavyakunder**
wishlist page

https://user-images.githubusercontent.com/81958043/154343083-08ab04f7-ef4f-4a89-9068-bf47d99cc508.mp4

### CODE REVIEW

File: css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  margin: 1rem;
+  border: 1px solid var(--light-grey-border);
+}
+.container-img {
+  position: relative;
+  width: 100%;
+  height: 65%;
+  overflow: hidden;
+}
+.imgg {
```
Comment (dev-enforced) [inline css/wishlist.css]: Suggestion:
Can keep it simply as .img or anything as per your convenience

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+      integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="https://robocon-ui.netlify.app/styles.css" />
+    <link rel="stylesheet" href="../css/wishlist.css" />
+
+    <title>My Project</title>
```
Comment (dev-enforced) [inline pages/wishlist.html]: Can rename it to Wishlist !!

File: css/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+.rating i {
+  color: #ecc722;
```
Comment (dev-enforced) [inline css/wishlist.css]: Suggestion: Can store it as a variable in wishlist.css as icon-color and use it!!

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !! Suggested some minor changes which you can do after merge!!The demonstration from photos also looks good!!

File: css/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+* {
+  box-sizing: border-box;
```
Comment (ZuberDunge) [inline css/wishlist.css]: You can shift this to reset.css

File: css/wishlist.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: var(--secondary-dark);
+}
+
+body {
```
Comment (ZuberDunge) [inline css/wishlist.css]: Giving flex to body tag, can be messy for user, try giving a class and then apply styling to that.

**Review by ZuberDunge — APPROVED** [review]
Added few suggestions, Apart from that, Everything looks great. Good to go from my side.

File: css/wishlist.css (hunk 1 above)
Comment (kavyakunder) [reply css/wishlist.css]: I changed classname to wishlist-img

File: pages/wishlist.html (hunk 2 above)
Comment (kavyakunder) [reply pages/wishlist.html]: done!

File: css/wishlist.css (hunk 3 above)
Comment (kavyakunder) [reply css/wishlist.css]: done!

File: css/wishlist.css (hunk 4 above)
Comment (kavyakunder) [reply css/wishlist.css]: I removed it as it was unnecessary.

File: css/wishlist.css (hunk 5 above)
Comment (kavyakunder) [reply css/wishlist.css]: removed it!

**Pull request merged**